crates/cli/src/bin/wasm-bindgen-test-runner/node.rs

```diff
@@ -30,7 +30,7 @@ wrap("info");
 wrap("warn");
 wrap("error");
 
-cx = new wasm.WasmBindgenTestContext();
+const cx = new wasm.WasmBindgenTestContext();
```

Comment:
No, that is required for deno, because deno implementation uses a part of the node implementation.

Reply:
I think it would be good if you explain what you did exactly and why you did it in the OP or comments.

(maybe its more obvious for the actual reviewer)

Reply:
To be honest I don't remember all the details.

But by looking at the diffs, I don't think I made any decisions, I tried to follow the original intentions, I just added missing pieces and changed the order of some code to get it working.

I agree its hard to understand because the errors surfaced mainly in the generated run.js file, in this case, I was getting a variable missing error.

On another the variable was used before it was declared for instance...

Reply:
I think the old version didn't work in Deno because it has [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) enabled by default, which doesn't allow implicitly declaring global variables like that.